### PR TITLE
[IMP] Remove nowrap from Ellipsis class

### PR DESCRIPTION
**Pull request by S-mardii**
Changing CSS for the class of Ellipsis and clean code in the file.

### CODE REVIEW

File: less/_mobile.less [hunk 1]
```diff
@@ -297,170 +302,186 @@
...
 
-	.interactive-map .map-container.hide-on-mobile-dialog{
-		max-height: 300px;
-	}
+  .interactive-map .map-container.hide-on-mobile-dialog {
+    max-height: 300px;
+  }
 }
 
 .ellipsis {
```
Comment (S-mardii) [inline less/_mobile.less:480]: The main part of the change is on this class only.

**Comment by EricSoroos** [comment]
@S-mardii What node are you using to run gulp on the less files?

**Comment by S-mardii** [comment]
@EricSoroos Are you referring to the version of Node JS? If yes, for this theme, we are using Gulp 3.9.1 (the latest version of Gulp is 4.0.2) based on the `package.json`. I am not sure which Node version is using, but it should be lower than v10.

As far as I know, Gulp 3 is not really supported with v10+ of Node.

**Comment by EricSoroos** [comment]
Thanks, it looks like it works on 8.

**Comment by S-mardii** [comment]
I tried Node 11.15.0 and it works too.

**Pull request merged**